app/mailers/makers_mailer.rb

```diff
@@ -3,6 +3,6 @@ class MakersMailer < ApplicationMailer
 
   def cancellation_email(reservation)
     @reservation = reservation
-    mail(to: @reservation.user.email, subject: 'Tu reservación ha sido cancelada.')
+    mail(to: @reservation.user.email, subject: 'Tu reservación ha sido rechazada.')
```

Comment:
I checked and this method is called whenever a reservation is cancelled, not only when it is cancelled by and admin. We should change back to 'cancelada', because the user will be notified even when they cancelled it themselves.